content/posts/text-to-cad-blender-addon.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: 'Text-to-CAD Add-on For Blender'
+excerpt: |
+    Introducing our Blender add-on utilizing our Text-to-CAD API.
+coverImage: '/documentation-assets/zoo-banner.png'
+date: '2024-03-15T12:00:00Z'
+author:
+    name: Ryan Barton
+    picture: '/documentation-assets/ryanbarton.jpg'
+ogImage:
+    url: 'documentation-assets/blender-addon/text-to-cad-blender-addon.jpg'
+---
+
+Today we are introducing our [Text-to-CAD add-on for Blender](https://github.com/KittyCAD/text-to-cad-blender-addon). In the world of digital design and 3D modeling, the bridge between conceptualization and realization is crucial. Powered by [Zoo.dev](https://zoo.dev/)'s API, this add-on allows users to generate CAD models directly from text descriptions.
+### What is Text-to-CAD?
+So glad you asked! Our CEO Jessie Frazelle has actually already written a nice [blog post](/blog/introducing-text-to-cad) describing our Text-to-CAD technology.
+
+TL;DR Text-to-CAD transforms text into editable CAD models, distinct from 3D meshes by generating B-Rep surfaces for compatibility with CAD programs.
```

Comment:
This is the only paragraph that I think could be extended if you want to. If this is meant to be read by Blender users who might not know what B-Rep's are or why they can be more powerful than generated meshes. I made an issue to add it to the glossary so that in future people can just link to the term.

Reply:
I didn't want to go too in depth with B-Rep explanations since I felt like that was covered in Jess's blog article, I just wanted this blog post to be about using our API and how it can easily be integrated with other modeling software